Amplify/Categories/Auth/Request/AuthChangePasswordRequest.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct AuthChangePasswordRequest: AmplifyOperationRequest {
+
+    public let currentPassword: String
+
+    public let newPassword: String
+
+    public var options: Options
+
+    public init(currentPassword: String,
+                newPassword: String,
+                options: Options) {
+        self.currentPassword = currentPassword
+        self.newPassword = newPassword
+        self.options = options
+    }
```

Comment:
Since these are structs, do we need to include the inits? (applied to all of these *Request/Result struct/extensions)

Reply:
ah! thanks!